server/src/server.ts

```diff
@@ -57,7 +57,7 @@ connection.onInitialized(() => {
   }
 
   connection.client.register(DidChangeWatchedFilesNotification.type, {
-    watchers: [{ globPattern: `**/app/javascript/**/*` }],
+    watchers: [{ globPattern: `**/app/javascript/**/*` }, { globPattern: `**/resources/js/**/*` }],
```

Comment:
We probably just want to dynamically construct these globs based off of `this.project.controllerRoots`:

Something like:
```suggestion
    watchers: this.project.controllerRoots(root => `**/${root}/**/*`)
```